ParabolInc/parabol
# feat(teams-limit): add addFeatureFlagToOrg mutation and support teams limit feature flag

Fixes #7519

- Added a new mutation `addFeatureFlagToOrg` that allows to add a feature flag to the `Organization` type
- Removed `teamsLimit` feature flag from the `User` type and added it to the `Organization` type

## Testing scenarios

- Go to http://localhost:3000/admin/graphql
- Select private schema
- Run the following mutation putting your org ids

```
mutation {
  addFeatureFlagToOrg(orgIds: ["ivQxFEZ7tC", "ijVADO81eo"], flag: teamsLimit) {
  	...on ErrorPayload {
      error {
        title
        message
      }
    }
    ...on AddFeatureFlagToOrgSuccess {
      updatedOrganizations {
        id
        featureFlags {
          teamsLimit
        }
      }
  	}
  }
}
```

- See that result returned correctly
